### PR TITLE
Make app look good

### PR DESCRIPTION
**Pull request by noahtallen**
### Test:
- run app
- test that every screen looks a lot better (e.g. splash screen, login, logo, buttons, colors, fab)
- make sure it don't crash

### CODE REVIEW

**Comment by lukemeierj** [comment]
Looks really good!  

I think the bio text could use some padding
Screen Shot 2019-03-21 at 12 55 18 PM

Solve with `bioText.textContainerInset = UIEdgeInsets(top: 12, left: 7, bottom: 12, right: 7)`:
Screen Shot 2019-03-21 at 12 59 04 PM
Screen Shot 2019-03-21 at 1 00 53 PM
Works on 5s as well

---

 Login screen doesn't format well on the 5s.  FYI, but probably not a big deal
Screen Shot 2019-03-21 at 1 00 34 PM

Works OK on other devices like iPhone 8
Screen Shot 2019-03-21 at 1 05 43 PM

**Comment by lukemeierj** [comment]
What do you think about purple topics?  It would be a nice way of differentiating a `BuddyButton` from a non-actionable data item.  I think it might be less pretty, but I think we should find some way of distinguishing actionable buttons, and this could do for now.

Screen Shot 2019-03-21 at 1 32 38 PM
Screen Shot 2019-03-21 at 1 32 13 PM

File: Buddies/Onboarding/SignUpInfoVC.swift [hunk 1]
```diff
@@ -63,6 +65,7 @@ class SignUpInfoVC: LoginBase, UIImagePickerControllerDelegate, UINavigationCont
         // Do any additional setup after loading the view.
         bioText.delegate = self
         bioText.textColor = UIColor.lightGray
+        bioText.layer.cornerRadius = 5
```
Comment (lukemeierj) [inline Buddies/Onboarding/SignUpInfoVC.swift]: I think the bio-text field looks nicer with a little padding.  

`bioText.textContainerInset = UIEdgeInsets(top: 12, left: 7, bottom: 12, right: 7)`

**Review by lukemeierj — CHANGES_REQUESTED** [review]

**Comment by lukemeierj** [comment]
I've been asking around about our logo material.  I'll put some of the feedback here.

Screen Shot 2019-03-21 at 1 39 03 PM

Some of the suggestions/notes I received:
* Consider avoiding the eyes going out past the smiley
* The colors don't fit great with smiley faces.  The milky white eyes aren't normal for a smiley drawing.  People mentioned something about a blue background being weird, and yellow fitting more with smileys.
* The eyes are too big
* I tried rounding the edges if the smile, and people liked the current square smile more.
* Looks kind of buggish
* Suggested using crescents instead of big moons
Screen Shot 2019-03-21 at 1 44 21 PM

I know a lot you guys have gotten a lot of positive feedback.  I've mostly gotten indifference or critique.  I'm not sure that means we have to do anything about it now, but I think we should keep an open mind about it.

**Comment by lukemeierj** [comment]
As for testing the PR, it tests super well! 🎉 

Can confirm that everything looks so much better and there are no extra bugs I can tell from this.  

I'm really excited to see this in master.  This is the point where the app is really becoming a thing, I'm really excited.  Thanks for putting your color senses to good use, @noahtallen  and @JakeThurman ! Looks great!

**Comment by JakeThurman** [comment]
I agree with all of Luke's feedback. Although
- I literally don't care either way about the smiley face changes, either is fine.


Also The purple topics is a better choice for distinguishing - I agree. Although It's not my fav look, it seems worth it.

**Comment by noahtallen** [comment]
Hm. Definitely agree about a lot of stuff here. My one thing is I don’t think the purple topics look good, which I think is the goal of the PR. I like that they’re more distinguished, but I think they look worse. Hm. Maybe we should find a different color?

**Comment by JadaTBK** [comment]
I think the crescent eyes on the smiley are a lot better; regarding the topic colors vs. button color, i think having a more "purple" but still darker color for the button would work better than the current blue. I know luke mentioned people thinking yellow would be a more appropriate color related to a smiley face. Do we wanna think about shifting from purple to yellow?

**Comment by noahtallen** [comment]
Jake and I did look into some Yellow colors, but I don't really like yellow as a primary color, personally. maybe as an alt?

**Comment by JadaTBK** [comment]
Yeah i think yellow would be good as an alt, would we want to have like orange be the primary color? To keep with the color family.

**Comment by noahtallen** [comment]
i personally really enjoy the deep purple as the primary color

**Comment by lukemeierj** [comment]
I think a slightly less dark primary color could be good.  The contrast between the bright whites and the very dark blue looks a little unfinished to me.  The apps I use usually have primaries that contrast less with the base white, and then occasionally dark, high contrast controls.

**Comment by noahtallen** [comment]
@lukemeierj @JakeThurman @gyurisic This could use another look. New changes:
1. Topics in View Activity look different and aren't buttons any more
2. Topic cards on the topic tab look different
3. Topics on pick topics for create activity look different
4. Tint color is now programmatically set globally for the app to the "Blue" color. This means that even things like the text cursor and action buttons on alerts show up in the right color. As a result, the navigation buttons are now "blue" rather than purple. I think it looks more consistent and better
6. Changed the destructive color back to red, so we either need to be ok with that or come up with something better than the maroon we had before
7. Switched smileys out to the crescent ones. 
8. Implemented the padding thing Luke suggested
9. Renamed the `ControlColors` class to `Theme` because I wanted to put the border radius constant in there


Todo(?):
- [ ] Better style for filter button on search bar? @lukemeierj this is something you have thoughts about; I don't think we reached any conclusions about it

**Review by lukemeierj — APPROVED** [review]

**Pull request merged**